src/views/Shift.vue

```diff
@@ -71,13 +80,20 @@ export default {
       this.order = new Order();
     },
     reset: function () {
+      if (this.orderBeingUpdated) {
+        return
+      }
+      clearTimeout(this.updateOrderToServer);
+      this.updateOrderToServer = null;
```

Comment:
Is setting to null necessary at all?

Reply:
It is if you want to check elsewhere if a timer is running or not (according to google)